sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/message.py

```diff
@@ -23,7 +23,7 @@ class MessageDict(TypedDict):  # needed for use with spread operator
         message_annotations: Optional[Dict[Union[str, bytes], Any]]
         properties: Optional["Properties"]
         application_properties: Optional[Dict[Union[str, bytes], Any]]
-        data: Optional[bytes]
+        data: Optional[Union[bytes, Iterable[bytes]]]
```

Comment:
Im wondering if we have to change the test to send in bytes versus an iterable of bytes. While creating a messaging are there cases when we expect a list of bytes ?

Reply:
We can change the test, but we also need to change this type. It should match with the public API of the data type here: https://github.com/Azure/azure-sdk-for-python/blob/b8c2bb5a85232b5e448c332806cf5f7fc943a4f4/sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py#L241